### DISCUSSION TITLE
`<DropdownMenu.Item>` as `<button type="submit">` doesn't submit the form

### DISCUSSION THREAD

**Discussion by tavoyne** [radix-ui/primitives#2071] [Help] [answered]
```tsx
<DropdownMenu.Item asChild>
  <button form="123" type="submit">
    Submit
  </button>
</DropdownMenu.Item>
```

When the button is clicked, the form does not get submitted as expected. However, if the button is triggered by pressing the enter key, or if the `onSelect` prop of the `<DropdownMenu.Item>` is set to `(evt) => { evt.preventDefault(); }`, then the form is submitted (though it prevents the content from being closed).

**Comment by joaom00** [comment, marked as answer]
I think the problem is exactly that, the submit event not bubble up to form because the button is unmounted when content closes
